x-pack/plugins/security_solution/server/lib/telemetry/filters.ts

```diff
@@ -0,0 +1,163 @@
+/*
```

Comment:
This module is responsible for filtering events from queried documents before forwarding them to the sender.